protocol/synthetix/contracts/modules/core/RewardsManagerModule.sol

```diff
@@ -242,6 +242,10 @@ contract RewardsManagerModule is IRewardsManagerModule {
         }
         pool.vaults[collateralType].rewards[rewardId].distributor = IRewardDistributor(address(0));
 
+        RewardDistribution.Data storage reward = pool.vaults[collateralType].rewards[rewardId];
+
+        reward.resetEntry();
```

Comment:
wondering why I don't have to specify an argument here?

Reply:
It's because we're using it as a library: https://www.tutorialspoint.com/solidity/solidity_libraries.htm

Basically, once you use the `using` keyword, all library functions in the file don't need the first argument.